decentraland/unity-renderer
# Feat: UI custom gradients

## What does this PR change?



This PR adds the required shader and script to implement custom gradients into UI elements. It is also applied into the top bar of the main menu.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/UI-Custom-Gradients
2. Open the main menu
3. Verify the movement of the gradient on the top bar

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
